jenkinsapi_tests/unittests/test_plugins.py

```diff
@@ -206,13 +214,18 @@ def test_install_plugin_good_input(self, _post, _wait,
 
     @mock.patch.object(Plugins, '_poll')
     @mock.patch.object(Plugins, 'plugin_version_already_installed')
+    @mock.patch.object(Plugins, 'restart_required',
+                       new_callable=mock.mock.PropertyMock)
     @mock.patch.object(Plugins, '_wait_until_plugin_installed')
     @mock.patch.object(Requester, 'post_xml_and_confirm_status')
     @mock.patch.object(Jenkins, 'safe_restart')
-    def test_install_plugins_good_input_no_restart(self, _restart, _post,
-                                                   _wait, already_installed,
-                                                   _poll_plugins):
+    def test_install_plugins_good_input_no_restart_required(self, _restart,
```

Comment:
Is there a reason these tests (I just changed the method names) are written in test_plugins.py instead of test_jenkins.py? The functions that are called, are from the `Jenkins` class, not the `Plugins` class..

Reply:
The tests are organized per functional area and not by class. This makes easier to find appropriate test and run tests if only specific functionality have been changed.